### PR TITLE
Enable dedicated f439 cmsis core dependency.

### PR DESCRIPTION
**Pull request by niklas-arm**
The title says it all.

### CODE REVIEW

File: module.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
       "cmsis-core-stm32f429xi": "^1.0.0"
     },
     "stm32f439zi": {
-      "cmsis-core-stm32f429xi": "^1.0.0"
+      "cmsis-core-st-stm32f439zi": "~0.0.0"
```
Comment (0xc0170) [inline module.json]: Why are we using -st prefix there? Majority of cmsis-core modules don't do this

Comment (niklas-arm) [inline module.json]: We're so good at naming things consistently.
Sure, we could fix this, but here are plenty of other repos that do not adhere to a coherent naming scheme…
¯\_(ツ)_/¯

Comment (0xc0170) [inline module.json]: Yes, we are ! :/ 
This module is still private, lets fix it.

**Comment by niklas-arm** [comment]
We will use #21.

**Pull request closed without merging**